payments-core/src/main/java/com/stripe/android/PaymentBrowserAuthStarter.kt

```diff
@@ -16,19 +16,15 @@ internal interface PaymentBrowserAuthStarter :
     AuthActivityStarter<PaymentBrowserAuthContract.Args> {
     class Legacy(
         private val host: AuthActivityStarterHost,
-        private val hasCompatibleBrowser: Boolean,
         private val defaultReturnUrl: DefaultReturnUrl
```

Comment:
can we pass `Context` here and calculate defaultReturnUrl within? exposing this implies defaultReturnUrl is some overridable parameter while it should always be constant

Reply:
I'd prefer to do this in a separate PR